modules/persistent-storage-csi-gcp-filestore-nfs-export-options.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_csi-google_cloud_file.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-gcp-filestore-nfs-export-options_{context}"]
+= NFS export options
+
+Network File System (NFS) sets file share permissions when creating a storage class. Using NFS export options propagate these options to the server for the Filestore instance.
+
+.Prerequisites
+* Access to the cluster as a user with the cluster-admin role.
+
+* The {gcp-short} Filestore CSI Driver Operator and {gcp-short} Filestore CSI driver installed.
+
+.Procedure
+
+. Create a storage class using a file similar to the following sample YAML file:
++
+[NOTE]
+====
+For more information about creating a storage class, see Section _Creating a storage class for GCP Filestore Operator_.
+====
++
+.Example storage class YAML file with NFS export options
+[source,yaml]
+----
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+ name: SC-name
+provisioner: filestore.csi.storage.gke.io
+parameters:
+ connect-mode: DIRECT_PEERING
+ network: project-network
+ nfs-export-options-on-create: '[ <1>
+   {
+     "accessMode": "READ_WRITE", <2>
```

Comment:
Looking at upstream code this parameter seems to accept two more options (`AnonGid`, `AnonUid`): https://github.com/kubernetes-sigs/gcp-filestore-csi-driver/pull/737/files#diff-6ea56a6be0f530a722fa9a5e04c47bbacd09301411b5ab5f08ccd220bf49f83cR53-R59

It's also in the upstream doc (which is the source for this PR I suppose?): https://cloud.google.com/filestore/docs/reference/rest/v1beta1/NfsExportOptions

Reply:
Done.